Run.py

```diff
@@ -1,7 +1,10 @@
 import pytest
+from Application import app, db # Ensure these imports point to your initialized app and db
```

Comment:
This causes a keyerror because when app or db is imported it runs Application/__innit__.py and it attempts to access os.environ["TESTING"].